packages/seo-store/src/analysis/slice/config.js

```diff
@@ -35,6 +39,7 @@ export const configSelectors = {
 	selectAnalysisType: state => get( state, "analysis.config.analysisType" ),
 	selectIsSeoAnalysisActive: state => get( state, "analysis.config.isSeoActive" ),
 	selectIsReadabilityAnalysisActive: state => get( state, "analysis.config.isReadabilityActive" ),
+	shouldApplyCornerstoneAnalysis: state => get( state, "analysis.config.shouldApplyCornerstoneAnalysis" ),
```

Comment:
Should the selector name start with "select" per [Redux recommendation](https://redux.js.org/usage/deriving-data-selectors#:~:text=A%20selector%20function%20can%20have%20any%20name%20you%20want.%20However%2C%20we%20recommend%20prefixing%20selector%20function%20names%20with%20the%20word%20select%20combined%20with%20a%20description%20of%20the%20value%20being%20selected.%20Typical%20examples%20of%20this%20would%20look%20like%20selectTodoById%2C%20selectFilteredTodos%2C%20and%20selectVisibleTodos.)?